.github/workflows/prettify-code.yml

```diff
@@ -0,0 +1,20 @@
+name: "Run prettier"
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Prettify code
+        uses: actionsx/prettier@v2
+        with: 
+          args: --check
```

Comment:
you need a .

Reply:
https://github.com/rutajdash/prettier-cli-action just found this action too. it might just be better. seems simpler